src/routes/Logout/components/LogoutPage/LogoutPage.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { useFirebaseApp } from 'reactfire';
+import { useHistory } from 'react-router-dom';
+import {
+  makeStyles,
+  Container,
+  Typography,
+  Paper,
+  LinearProgress,
+} from '@material-ui/core';
+import { Helmet } from 'react-helmet';
+import styles from './LogoutPage.styles';
+
+const useStyles = makeStyles(styles);
+
+function LogoutPage() {
+  const classes = useStyles();
+  const firebase = useFirebaseApp();
```

Comment:
You can use `const auth = useAuth()` then later `auth.signOut`

Reply:
Good point.  Adjusted.